src/main/java/org/jabref/logic/openoffice/action/EditSeparate.java

```diff
@@ -38,7 +39,7 @@ public static boolean separateCitations(XTextDocument doc,
             NotRemoveableException,
             PropertyVetoException,
             WrappedTargetException,
-            com.sun.star.lang.IllegalArgumentException {
+            IllegalArgumentException {
```

Comment:
I only find this dangerous because it's easy to mistake it for java's own one...